docs/src/main/asciidoc/cache.adoc

```diff
@@ -484,5 +490,87 @@ public class CachedService {
     }
 }
 ----
-<1> This method can be used to force a refresh of the cache entry corresponding to the given key. 
+<1> This method can be used to force a refresh of the cache entry corresponding to the given key.
 <2> This method will invalidate all entries from the `foo` and `bar` caches with a single call.
+
+[#negative-cache]
+== Negative chaching and nulls
+
+Sometimes one wants to cache the results of an (expensive) remote call.
+With all remote  calls, they may fail, in which case one may not want to store the bad results.
```

Comment:
This sentence is confusing, maybe reword a bit?

Reply:
Yes, you are right. Thanks for the suggestion